packages/web-runtime/src/index.ts

```diff
@@ -60,8 +62,31 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
   })
   const themePromise = announceTheme({ store, app, designSystem, runtimeConfiguration })
   await Promise.all([applicationsPromise, themePromise])
+  const customTranslations = {}
+
+  for (const customTranslation of configurationManager.customTranslations) {
```

Comment:
Could you please move fetching and collecting/merging the custom translations into its own method? Something like `fetchCustomTranslations` which then returns the `customTranslations` object. Otherwise this is an abstraction level mismatch. All the other code in this `bootstrapApp` method is on a higher abstraction level.